chapter5/tests/t3/terraform/_modules/container/variables.tf

```
@@ -0,0 +1,65 @@
+variable "container_image" {
+  description = "Value of the name for the Docker container"
```

```suggestion
  description = "Docker image name"
```

P.S. let's add a separate variable `container_image_tag`, so in you main.tf you will have
```
resource "docker_image" "container" {
  name         = "${var.container_image}:${container_image_tag}"
  keep_locally = var.container_image_keep_locally
}
```

---

done